### PR TITLE
Move all language installations to be managed by ASDF

### PR DESCRIPTION
**Pull request by gjastrab**

### CODE REVIEW

File: laptop.zsh [hunk 1]
```diff
@@ -67,21 +67,11 @@ brew "pre-commit"
 # GitHub
 brew "gh"
 
-# Programming languages and package managers
-brew "node"
-brew "yarn"
-brew "rebar"
-
 brew "hashicorp/tap/vault"
-cask "graphiql"
```
Comment (gjastrab) [inline laptop.zsh:76]: This line no longer worked when running through this with @ejdelsztejn 

She ran `brew cask install graphiql` and it worked but mentioned this is deprecated and that `brew install --cask graphiql` is now recommended, however I didn't see documentation for how to properly specify this in a Brewfile syntax, so just moved that outside of the Brewfile way of triggering.

**Comment by gjastrab** [comment]
When onboarding @ejdelsztejn yesterday we used this path and all worked for her

File: asdf/elm.zsh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/zsh
+
+echo "\n\nInstalling Elm…"
+
+yarn global add elm --prefix /usr/local
+yarn global add elm-format --prefix /usr/local
+yarn global add elm-oracle --prefix /usr/local
```
Comment (hritchie) [inline asdf/elm.zsh:7]: should we do elm-test, elm-analyse, elm-review here as well?

Comment (thaterikperson) [reply asdf/elm.zsh:7]: I typically run those that Heath mentioned from `assets/node_modules/.bin`. Which raises the question, do we need any at `/usr/local`? Perhaps that's where code editors look for them?

Comment (gjastrab) [reply asdf/elm.zsh:7]: My [language server](https://github.com/neovim/nvim-lspconfig#elmls) expects these to be on the global path, so this code that was previously already in the repo (just moved out from the initial laptop script) made sense to me.

File: laptop.zsh (hunk 1 above)
Comment (gmcintire) [reply laptop.zsh:76]: It looks like this should be fixed in homebrew soon, the changes are already here: https://github.com/Homebrew/homebrew-bundle/blob/master/lib/bundle/cask_installer.rb

File: asdf/elm.zsh (hunk 2 above)
Comment (gmcintire) [reply asdf/elm.zsh:7]: My setup also expects these to be in the $PATH somewhere.

**Review by gmcintire — APPROVED** [review]

File: asdf/elm.zsh (hunk 2 above)
Comment (gmcintire) [reply asdf/elm.zsh:7]: Alternatively, asdf also provides support for installing elm

**Pull request merged**